packages/generator-wcfactory/generators/factory/templates/scripts/postinstall.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs-extra')
+const path = require('path')
+const pkgJson = require('../package.json')
+
+// get the factory npm org name as it is stored in node_modules
+const factoryName = pkgJson.wcfactory.orgNpm
+/**
+ * @todo this should be dynaimc probably
+ */
+// get elements directory in the
+const elementsDir = path.join(__dirname, '../elements')
+
+// loop through the elements directory and clean up the node_modules dir
+fs.readdir(elementsDir, (err, folders) => {
+  // move on to the next thing if there is an error
+  if (err) return
+  folders.forEach(element => {
+    const elementDir = path.join(elementsDir, element)
+
+      // re-symlink the node_modules dir
+      // make sure that this is a directory
+      const nodeModulesDir = path.join(elementDir, 'node_modules')
+      // check if this element has a node modules dir in it
+      if (fs.existsSync(nodeModulesDir)) {
```

Comment:
`fs.exists` has been deprecated, `fs.access` or `fs.stat` would be a safer alternative

Reply:
I don't think those are in fs-extra yet?

Reply:
https://github.com/jprichardson/node-fs-extra/blob/8e0879110fbdd597c36602fe3b81ef03a4b3ec7a/lib/fs/index.js#L8